test/parallel/parallel.status

```diff
@@ -7,6 +7,8 @@ prefix parallel
 [true] # This section applies to all platforms
 # Postmortem debugging data is prone to accidental removal during V8 updates.
 test-postmortem-metadata: PASS,FLAKY
+# Linux perf is prone to breakage during V8 updates.
+test-linux-perf: PASS,FLAKY
```

Comment:
Is this the case here? I'm a bit worried that it will make it less likely to be caught early on.  I'm not sure it is as likely to be broken as the postmortem data to warrant this.

Reply:
We do know if it breaks on [nodejs/node-v8](https://github.com/nodejs/node-v8), that's how we know when there are postmortem metadata changes today.

Removing flakiness status from this test would make it a blocker on V8 updates if Linux perf stops to work again. I can remove it if that's what we want.

Reply:
Removed flakiness status from this test. It's easy to add it back if we want to.

Reply:
@mmarchini seems like the right answer, we can always change if it is a problem.